src/humanize/filesize.py

```diff
@@ -15,7 +15,20 @@ def naturalsize(value, binary=False, gnu=False, format="%.1f"):
     By default, decimal suffixes (kB, MB) are used.
 
     Non-GNU modes are compatible with jinja2's `filesizeformat` filter.
+    Examples:
```

Comment:
Please check the lint errors on the CI, newlines are needed before all "Examples" sections:

```suggestion

    Examples:
```

To run lint locally:

```sh
pip install -U tox
tox -e lint
```

Or:
```sh
pip install -U pre-commit
pre-commit install # Optional, to enable for commits
pre-commit run --all-files
```